common/src/main/java/com/wynntils/core/chat/tabs/ChatTabModel.java

```diff
@@ -48,7 +49,7 @@ public static void onScreenOpened(ScreenOpenedEvent event) {
         if (!(event.getScreen() instanceof ChatScreen chatScreen)) return;
         if (focusedTab == null || focusedTab.getAutoCommand() == null) return;
 
-        chatScreen.insertText(focusedTab.getAutoCommand(), true);
+        ((ChatScreenInvoker) chatScreen).invokeInsertText(focusedTab.getAutoCommand(), true);
```

Comment:
Since this is a bit ugly (and can hopefully be reverted in the future if this is fixed upstream), maybe you can wrap this in a static helper function `insertText(ChatScreen chatScreen, String text)` or maybe even better name it `replaceChatText` since that is what this is actually doing.

Reply:
In idea, this folds itself nicely.

Reply:
Actually, it does not. But I dont really think we should change this.

Reply:
Not even to rename it `replaceChatText`? (I really dislike the vanilla design of having a boolean which turns an "insert" into a "replace" -- I needed to go read the code to understand why this was not broken)

Reply:
Okay, Ill do it

Reply:
Thanks!